client/signup/steps/passwordless/index.jsx

```diff
@@ -0,0 +1,261 @@
+import FormLabel from 'components/forms/form-label';
+import FormTextInput from 'components/forms/form-text-input';
+import LoggedOutForm from 'components/logged-out-form';
+import LoggedOutFormFooter from 'components/logged-out-form/footer';
+import Button from 'components/button';
+import { createPasswordlessUser, verifyPasswordlessUser } from 'lib/signup/step-actions';
+import Notice from 'components/notice';
+
+import FormStateStore from 'lib/form-state';
+import createFormStore from 'lib/form-state/store';
+const { getFieldValue } = FormStateStore;
+
+export class PasswordlessStep extends Component {
+	static propTypes = {
+		flowName: PropTypes.string,
+		translate: PropTypes.func,
+	};
+
+	static defaultProps = {
+		translate: identity,
+	};
+
+	state = {
+		errorMessages: null,
+		noticeMessage: '',
+		noticeStatus: '',
+		submitting: false,
+		showVerificationCode: false,
+	};
+
+	UNSAFE_componentWillMount() {
+		this.formStore = createFormStore( {
+			syncInitialize: {
+				fieldNames: [ 'email', 'code' ],
+			},
+		} );
+	}
+
+	handleFieldChange = event => {
+		event.preventDefault();
+
+		this.setState( {
+			errorMessages: null,
+		} );
+
+		this.formStore.handleFieldChange( {
+			name: event.target.name,
+			value: event.target.value,
+		} );
+	};
+
+	createUser = event => {
+		event.preventDefault();
+		const data = { email: getFieldValue( this.formStore.get(), 'email' ) };
+
+		this.setState( {
+			submitting: true,
+		} );
+
+		createPasswordlessUser( this.handleUserCreationRequest, null, data );
+	};
+
+	handleUserCreationRequest = ( error, response ) => {
+		if ( error ) {
+			this.setState( {
+				errorMessages: [ error.message ],
+				submitting: false,
+			} );
+
+			return;
+		}
+
+		this.setState( {
+			errorMessages: null,
+			noticeMessage: response && response.message,
+			noticeStatus: response && response.warning ? 'is-warning' : 'is-info',
+			showVerificationCode: true,
+			submitting: false,
+		} );
+	};
+
+	verifyUser = event => {
+		event.preventDefault();
+		const data = {
+			email: getFieldValue( this.formStore.get(), 'email' ),
+			code: getFieldValue( this.formStore.get(), 'code' ),
+		};
+
+		this.setState( {
+			errorMessages: null,
+			submitting: true,
+		} );
+
+		verifyPasswordlessUser( this.handleUserVerificationRequest, null, data );
+	};
+
+	handleUserVerificationRequest = ( error, providedDependencies ) => {
+		if ( error ) {
+			this.setState( {
+				errorMessages: [ error && error.message ],
+				submitting: false,
+			} );
+
+			return;
+		}
+
+		this.submitStep( providedDependencies );
+	};
+
+	submitStep = providedDependencies => {
+		const { flowName, stepName } = this.props;
+
+		SignupActions.submitSignupStep(
+			{
+				flowName,
+				stepName,
+			},
+			providedDependencies
+		);
+
+		this.props.goToNextStep();
+	};
+
+	createUserButtonText() {
+		const { translate } = this.props;
+
+		if ( this.state.submitting ) {
+			return translate( 'Creating Your account…' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **32** times:
`translate( 'Creating your account…' )` *ES Score: 8*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/passwordless-signup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
@scruffian can you use the existing capitalization so we don't create another new string?